packaging/files/enable.deb

```diff
@@ -1,2 +1,4 @@
-export GEM_PATH="/opt/ood/gems:${GEM_PATH:+:${GEM_PATH}}"
+export GEM_HOME="/opt/ood/gems"
```

Comment:
We'll have to check how this may impact app development.  Seems like when I develop apps and have gems in `GEM_HOME=~/.gems` OOD can recognize that.

Reply:
It should be fine because we do this with the SCL enable for RHEL systems:

```
export GEM_HOME="/opt/ood/ondemand/root/usr/share/gems/3.0"
```

This is just used by when nginx_stage and ood-portal-generator are run.  I think ~/.gems gets used if you tell gem to intall non-system gems.

Reply:
Actually I don't think GEM_HOME makes it into PUN.  At least it didn't used to until https://github.com/OSC/ondemand/commit/d4a9f611eaa708b44ff939dbf128e8c059fe9120 so maybe have to tweak that change to avoid GEM_HOME making it into PUN